Report/create_reports.R

```diff
@@ -44,9 +44,15 @@ predictions_cards = get_covidhub_predictions(forecasters,
     filter(!(incidence_period == "epiweek" & ahead > 4))
 
 predictions_cards = predictions_cards %>%
-    filter(!is.na(predictions_cards$target_end_date)) %>%
+    filter(!is.na(target_end_date)) %>%
     filter(target_end_date < today())
 
+# For hospitalizations, drop all US territories except Puerto Rico and the
+# Virgin Islands; HHS does not report data for these.
+territories <- c("as", "gu", "mp", "fm", "mh", "pw", "um")
+predictions_cards = predictions_cards %>%
+    filter(!(geo_value %in% territories), data_source != "hhs")
```

Comment:
I believe the comma acts as "and" rather than "or",  so we probably want "data_source == 'hhs'".